web/frontend/src/components/slam.vue

```diff
@@ -117,6 +117,30 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const fetchFeatureFlags = (name: string): Promise<{[key: string]: boolean}> => {
+  return new Promise((resolve, reject): void => {
+    const request = new commonApi.DoCommandRequest();
+    request.setName(name);
+    request.setCommand(Struct.fromJavaScript({ feature_flag: true }));
+    props.client.slamService.doCommand(
+      request,
+      new grpc.Metadata(),
+      (error: ServiceError|null, responseMessage: commonApi.DoCommandResponse|null) => {
+        if (error) {
+          if (error.code !== grpc.Code.Unimplemented) {
+            reject(error);
+            return;
+          }
+          resolve({});
+          return;
+
+        }
+        resolve(responseMessage!.getResult()?.toJavaScript() as {[key: string]: boolean});
```

Comment:
Can you please confirm that this does the right thing if viam-cartographer returns an empty map i.e. resolves to an empty js object?

Reply:
This returns an empty map and does not update the position if an empty map is returned, but just to note, that case should never happen in the current implementation because either a map is returned with the feature flags or an unimplemented error is returned

Reply:
Thanks for checking that! I mainly wanted to confirm that when we remove the feature flag later we won't cause this code to throw an exception.